Assets/MixedRealityToolkit.Examples/Experimental/Dialog/DialogExampleController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Experimental.Dialog;
+
+namespace Microsoft.MixedReality.Examples.Experimental.DialogTest
+{
+    public class DialogExampleController : MonoBehaviour
```

Comment:
Can you add a summary comment explaining what the component does and how it operates